xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -829,6 +1069,7 @@ final class DatabaseService {
             sql: """
             SELECT id, title
             FROM memo
+            WHERE substr(memo.slug, 1, 1) != '_'
```

Comment:
Might be worth a comment near each place we consider hidden slugs, since #705 will mean changing them all at once.